Src/BootCamp.Chapter/Computer/Ram.cs

```diff
@@ -2,5 +2,11 @@
 {
     public class Ram
     {
+        private readonly string _ram;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_ram' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
What is meant here. I used that variable in the constructor and later on for example the msfactory class I used it to set a contents to it ?